styles/main.header.scss

```diff
@@ -33,3 +33,114 @@
mZnsyXkoLUUsDYAWit6C0BnegNseLfWHNeYISnMGjAyRHFsjDYgElZgVaijjZuobGhnSzPkLT16h7oAH1RaZPHGuCWprXE//ELbdOiVw+4Or75P/3423XV0yee8PGulm3tIZyN917w8J7Bsa4Q4++1Lw+t9/kVWMXbykoOfzwsZ98n+XfvmTk3L45/7j0qklXTBghtTfRfUB7AcAaok+7lnkksxtAJZ+MB7EKsseWABFKkTKlJUBZuiVAGT184jwtAaLYyNiixAGqhYTdXxLCCI2Y66DNASq7NAfAzFspaIdLoQtpEoAyl7vgRgGzsSnC9RfQLECjPdk+Od0wIJNmVd1phh0Uwr9As/+8jUJUbaMA1KoO11JqYSsFMRvTKXtqqCCFkWbV/1FjBdz6F9pc4fpDZJD8lwtrscBdnMpP7KRbBVPHPJFJtzBQqqRXuilneVQqdKtM0w1e1Z6Xbni0R6XKa7XCs8hrZRVpZsNKRprH3YV8ESRfiZ3+0YWwHOikQEh34Vx3Dan+LxL+944dAy+A8/iskyenTOlkvhQNPwMa9mdGMAczaTgYqFaBYj8STTTT0F9bTUUzmL0xJXgYjPRM3ZGUugPQ+dEfxPKANH0H2FPRwSC+wT6zkjYSoeTGZrT4SQ9UH4VdZweAZWT3V9TEmtPsykiDu1B62IV1/lDsCLZruOKCyT3DVNM4qr6w8uLaAfmTbIeUd42zxvsvtbEXtABDr6rP1+UW5U6hL6qqMlYD7Y2u8rMecwu7SlDkajEalYJcUoz0sgINIAOOSqVmDAyKZTQZx0k9QueI19Iu4jWCv1pv7pH3G2EVIqWgiOx+A1YtSLnVQm8i9kJWAf1QKaqjc/AXKf/AVzMUAs848ctMniaqIi1UumpHUBnbhxnC/K27lB2EQKNWCX2H8ZMo07P4SPRrtNRaKjCETh3/Q3sVwmGwgmNKFCBmx1zNRJ3yru6XBPQwIHYMS+n9MbE/KLXS2kYDpXm5SvMeRPSgfoAHquti/5GQdnQGgkKZbZUvUG7nLh1/8ZJrx7WytYGiSKSouOYCRXjbjvnzx140f28gHA4U19SAHJp49rhuEL8AMFQLM5x4mHgIcVRVFLvG09dSHaxCHa0eq8NCMiehOAvLZ8WhkbiZehnNdlif/r6QE3BWf/RKjKCLU+ugSefAwtgy3QcM3j8ijVQKNy1PHbhJKVPW2EX9K1JTzs9in1fgTZtWo3dWtOnwu9hkb4s19YG3ffF7HD7KqE7WxeJwGV8BNH9O3yem0er6NqX6hJFzrtBa5Vr0BdrsAN8aBclZhbm6AO7L0OTPr1MKgXywb2tjMclppv3c6LpqGjOO/Sj0NkQLGa6zo0thCLtpq+vb0Ex0Vp4oDV4mlvVZPm3fdUsXXTXzJq3cbiPWdwg7f+UxQki+of1TzZZlW0eTgNLzRf74ppq5e65bsGc+Kbhy3No1DzYOm1BTxO5Wm8BM30pCsemD+xWXe1wFy6dMnfTkkfvWy9vlD1INYf42dErzmC2Lmkc8uj7sNzsKiqJkJLaHWYu9heQltLdQFND+Ob2F8NSA2tSZRt0aDImBMOYFdR5dhq2GMCMMdlKiUImI4tlk5VYMrZk4IccXCNbUUrukNKDo7BwhXliCedG/3J3IRnqOrZWwvTQtmn3X9suqzgmsTX7nxDltjK55dGoFO7xbVC2fNFBMg32BbqI2UA7IqqlKLTYWUcWNlMONFrXXY0Z7IJ0z3R4oSCmjo+2BaOCe1r/ShCUdnbHk5ehJbWpHoEzmKSqmTgZObatIWwIRUmDooSPQaoC672xLNwUaR7sCZbLATqLhB5/s+CPtCnRDl3kVwrwu6XleRel55cG8vMq8vF3n5aXzYiSvOp8ijorJnudD1LkgFj3ffEhe51w2HkQs2tOMiEOZDvkedUrmnPLAYu1lrbLScyqGOeUrc8rn0+Yrzik/Qovh0muVr84tSymq72WtVKR4Acu1hVS9ScKndjy8+PxLxl2K8DC9aqg3u/JjKbO691mKJVHgLowH9cybZRm82ebVodcjB0SxTqlnAbRCc5eCYNa2GZ2cn3ZvEOLmvFzqB1X5txda0OwMhRAIJs5HjPhXqzupQFZ81RshUmRQVhvjfUsyehY1qNE49IvQRi3p4KTp3OCkxGjSjhElSHkfGXjo3SD24sFun2OxCQHticTu7rx/uidSZj+kVBekLoFK5WxXXu0exAGzryK+n5CPD9PeSDSXmz27Ar7tpD0FjXh/Xcq/QhsjpYOWps6gJXyqUdJh9CrxOYXp0Pu0LsVUVuLDJ0yZkpoJxi4fh2etUOdCe83zqadhZSSWp+kjKt3MKbr9Z9HLGSfgwZav1Iap6WmqvdXmaE4xBUw55qCkD3Ok/XTKMK2Oli7TKivJ70qKfjvKFMQGWfYkTQKltegWO62d2qsxWwWXEsjLoo1V6clJUhYtqRIkvUHFXPTwisYS6nkGpYG+aGxnfE4fmNG3PzP7qt+F2CF/vvP6ByaMff2S4TcuX0/mtW45tOwL+Sgpu20ptvhayuatO7Z53JXDuMfXtzZOq8lfGaoa03ei/C+5+kn57M5WEiWSvO/Qgd27XlF4h58Idlw2WMLrOrsbFSLBQ9FERLHfgoURsN+qFfstJ9K9+VF9qvmRxNZFMJM/kaNYZTnhjF5IbSGjSZ86HZTaxNgYqU1bGFQSWX6hOxLXg/3VU8ekP/ZuafXSTIk/do5xRXUq7a9Ec5IwKnG5GpPwapP0EFqFLXKxb4ESuXE5aFQiW22zhD5wF+4vO1r4e7HZkuB00Jnmejv7LUn2VDZS175Ligupx95LFahFt/fSgEnjOtkxLNWDKXMO2RlzwC5RvvQcsjrn0K1VVIHaKoqOXnXlG6grv0vPKMmnoT70XntHoR7tsX/Udqo0e+8ixVrp3sycByKBueo80E8qpOdRoFX6nMM8vDAPr9LlEOeBfn6vuha5wnN0LVwOD50O1jYyUkFOxnQEDf3V7tNJu1h
```

Comment:
Rather than open up PRs to multiple different repos (`stellar/solar`, `stellar/solar-stellarorg`, `stellar/interstellar-core`) and coordinate their merge/deployment, I'm just slapping a bandaid over it so it looks pretty.